### PR TITLE
Updated env. var for release builds

### PR DESCRIPTION
**Pull request by mpepping**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -37,12 +37,12 @@ jobs:
...
-        run: printf 'TAG=' && curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' >> $GITHUB_ENV
+        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/TAG=\1/' >> $GITHUB_ENV
 
       -
         name: Build and push
         uses: docker/build-push-action@v2
         with:
           push: true
-          build-args: VERSION=${{ matrix.version }}
+          build-args: VERSION=${{ env.TAG }}
```
Comment (soinu) [inline .github/workflows/ci.yml:47]: This won't work because the TAG for master is latest. I guess we'll need two additional steps to set build arg vars

Comment (tabic) [inline .github/workflows/ci.yml]: This won't work, I guess we'll need additional steps to set the VERSION build arg or make the run above multiline

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -37,12 +37,12 @@ jobs:
       -
         name: Set tag for release
         if: ${{ matrix.version == 'release' }}
-        run: printf 'TAG=' && curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' >> $GITHUB_ENV
+        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/TAG=\1/' >> $GITHUB_ENV
```
Comment (tabic) [inline .github/workflows/ci.yml]:
```suggestion
        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's|.*"([^"]+)".*|TAG=docker.io/mpepping/cyberchef:\1|' >> $GITHUB_ENV
```
Sorry, this broke with my last PR

Comment (mpepping) [reply .github/workflows/ci.yml]: No worries! It's an opportunity to improve on our GitHub Actions skills right. :)

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mpepping) [reply .github/workflows/ci.yml]: Added `env.VERSION` containing the right values. I'll squash and push my commits again if this builds OK.

File: .github/workflows/ci.yml [hunk 3]
```diff
@@ -34,15 +34,25 @@ jobs:
...
       -
         name: Set tag for release
         if: ${{ matrix.version == 'release' }}
-        run: printf 'TAG=' && curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's/.*"([^"]+)".*/\1/' >> $GITHUB_ENV
+        run: curl -s https://api.github.com/repos/gchq/CyberChef/releases/latest | grep '"tag_name":' | sed -E 's#.*"([^"]+)".*#TAG=docker.io/mpepping/cyberchef:\1#' >> $GITHUB_ENV
+
+      -
+        name: Set version for release
+        if: ${{ matrix.version == 'release' }}
+        run: echo "VERSION=${TAG}" >> $GITHUB_ENV
```
Comment (tabic) [inline .github/workflows/ci.yml:50]:
```suggestion
        run: echo "VERSION=${TAG}" | sed 's/.*://' >> $GITHUB_ENV
```

**Pull request merged**